src/Nokia_LCD.cpp

```diff
@@ -133,8 +138,10 @@ bool Nokia_LCD::printCharacter(const unsigned char character) {
         return mY_cursor == 0;
     }
 
-    return draw(Nokia_LCD_Fonts::kDefault_font[character - 0x20],
+    bool out_of_bounds = draw(Nokia_LCD_Fonts::kDefault_font[character - 0x20],
                 Nokia_LCD_Fonts::kColumns_per_character, true);
+
+    return draw(inverted ? Nokia_LCD_Fonts::space : Nokia_LCD_Fonts::space, 1, false) || out_of_bounds;
```

Comment:
What does the space do exactly? Is it making sure that the last column in each displayed character will be blank?
If that is correct, isn't it better if we modify the fonts instead? Also, how do fonts look afterwards? I cannot try it right now, but do they look better? Can you share some photos?

Other than that, am I missing something with `inverted ? Nokia_LCD_Fonts::space : Nokia_LCD_Fonts::space` or should it just be replaced with `Nokia_LCD_Fonts::space`?

Reply:
The invert stuff look good btw, nice addition. :+1:

Reply:
inverted ?: was a mistake, thx for catching it!. Fixed!

Space separates chars in a line. Without this space, even non inverted text looks too tight.
With my inverted mode, it looks terrible.

I was thinking about adding this extra space in the font, but that will dramatically increase the size without the need. I think chars definition should not care about space between chars.

Anyway, you solution does not allow to change the font on the fly, like it is done in Adafruit's lib. Btw, did you think about adding setFont method?

Reply:
> I was thinking about adding this extra space in the font, but that will dramatically increase the size without the need. I think chars definition should not care about space between chars.

Fair enough, sounds reasonable.

> Space separates chars in a line. Without this space, even non inverted text looks too tight.
With my inverted mode, it looks terrible.

Would it be too much trouble to ask you to take a picture of a before and after pictures? Overall it sounds like a good idea, I just need to give the future developer of this an idea on why this change took place. :)

I'd also like if you could put a comment before the `return` statement indicating the intent, e.g. `// Separate the characters with a vertical line so they don't appear too close to each other` or something relevant. :+1: 

> Anyway, you solution does not allow to change the font on the fly, like it is done in Adafruit's lib. Btw, did you think about adding setFont method?

I **think** there's a space issue if we start adding more fonts and/or change them dynamically. However, it's been a very long time since I used or worked on this library, so I might be mistaken and the `setFont` function might have simply been removed because I did not need it at the time.